docs/connect.rst

```diff
@@ -99,6 +99,16 @@ You can get a PDO connection like this:
    authenticate as the CrateDB superuser, which is ``crate``. The superuser
    does not have a password, so you should omit the ``password`` argument.
 
+CrateDB Cloud
+=================
+
+If you are connecting to CrateDB Cloud you will need to enable the SSL mode 
+by setting the connection's ``CRATE_ATTR_SSL_MODE`` attribute. 
+See the section below for details on how to do this.
+
+If this is not configured you will see the following error message: 
+``cURL error 52: Empty reply from server``
+
```

Comment:
Thank you. I think, while already being at it, we should take the chance to add a full example here, how to use the `CRATE_ATTR_SSL_MODE` attribute properly in the context of one to three lines of code.

However, I will not bother you about it too much. What do you think about this idea?

Reply:
I added an example now, please check if it looks ok to you. 
I find that I need to use `PDOCrateDB` instead of `PDO` for the constants, should I update this everywhere else in the page?

Reply:
> I added an example now, please check if it looks ok to you.

I was looking for a "full connect" example, where one would also see a corresponding host name surrogate, similar like [the Python driver front page is doing it](https://cratedb.com/docs/python/en/latest/#db-api), but it's still okay now.

> I find that I need to use `PDOCrateDB` instead of `PDO` for the constants, should I update this everywhere else in the page?

That's bad, right? I vaguely remember something about it, and that it should be made different, or that it may have been different in a previous lifecycle of the driver?

It was probably changed when PHP was transitioning into a more strict governance of type-safety and namespacing policies, where we could no longer easily overlay/converge into `PDO`? Those are my blurry thoughts about this topic. Don't really know how to move on. Probably we need to change the documentation, yeah.

Reply:
> Probably we need to change the documentation, yeah.

However, it does not need to be squeezed into this PR. Feel free to send a subsequent update if you can afford the time.

Reply:
I've diverted this into GH-153.